composer.json

```diff
@@ -42,7 +42,7 @@
     "php": "^7.1 || ^8.0",
     "ext-dom": "*",
     "ext-simplexml": "*",
-    "guzzlehttp/guzzle": "^6.0",
+    "guzzlehttp/guzzle": "^6.0 || ^7.0",
```

Comment:
You should add a step in CI for php7.1 with `composer update --prefer-lowest...` to ensure guzzle 6 still work. I can add it if you're ok

Reply:
ok you can add